script/entrypoint.sh

```diff
@@ -64,22 +64,23 @@ if [ "$AIRFLOW__CORE__EXECUTOR" != "SequentialExecutor" ]; then
   wait_for_port "Postgres" "$POSTGRES_HOST" "$POSTGRES_PORT"
 fi
 
+
 if [ "$AIRFLOW__CORE__EXECUTOR" = "CeleryExecutor" ]; then
   AIRFLOW__CELERY__BROKER_URL="redis://$REDIS_PREFIX$REDIS_HOST:$REDIS_PORT/1"
   wait_for_port "Redis" "$REDIS_HOST" "$REDIS_PORT"
 fi
 
 case "$1" in
   webserver)
-    airflow initdb
+    airflow db init
     airflow connections --add --conn_id gcs --conn_type google_cloud_platorm --conn_extra "{\"extra__google_cloud_platform__key_path\":\"/usr/local/airflow/gcloud-secrets/$GCP_SERVICE_ACCOUNT.json\", \"extra__google_cloud_platform__project\": \"$GCP_PROJECT\", \"extra__google_cloud_platform__scope\": \"https://www.googleapis.com/auth/cloud-platform\"}"
     if [ "$AIRFLOW__CORE__EXECUTOR" = "LocalExecutor" ]; then
       # With the "Local" executor it should all run in one container.
       airflow scheduler &
     fi
     exec airflow webserver
     ;;
-  worker|scheduler)
+  celery|worker|scheduler)
```

Comment:
@fathom-chanmin 

I see your worker is getting lot's of 
`[2021-08-11 09:14:49,965: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.`

without this change airflow is setting up a pure celery instead of airflow-managed celery that can be setup with `airflow celery`


you need to apply this change here: https://github.com/medicode/docker-airflow/blob/master/script/entrypoint.sh#L82